cmd/neofs-adm/internal/modules/morph/initialize.go

```diff
@@ -28,6 +28,8 @@ type initializeContext struct {
 	// ConsensusAcc is used for retrieving committee address and verification script.
 	ConsensusAcc *wallet.Account
 	Wallets      []*wallet.Wallet
+	// ContractWallet is a wallet for providing contract group signature.
+	ContractWallet *wallet.Wallet
```

Comment:
It called `manifest wallet` in code above.

Reply:
Fixed.